test/dash/dash_parser_manifest_unit.js

```diff
@@ -1370,4 +1370,36 @@ describe('DashParser Manifest', () => {
     expect(textStream.roles).toEqual(['captions', 'foo']);
     expect(textStream.kind).toBe('caption');
   });
+
+  // Filter image adaptation sets, until we add thumbnail track support.
```

Comment:
Nit: If you change the name of the test isn't accurate (see comment below), this comment won't be necessary anymore.

Reply:
Removed!